src/components/personkort/PersonkortHeader/PersonkortHeader.tsx

```diff
@@ -87,7 +61,7 @@ const PersonkortHeader = () => {
             {formaterFnr(personident)}
             <CopyButton message={texts.copied} value={personident} />
           </StyledFnr>
-          <HeaderInfoStartDate startDate={startDate} />
+          <ExtraInfo />
```

Comment:
ExtraInfo er litt generelt, hva med noe som er beskrivende for innholdet. Feks SyketilfelleInfo, KortOmSykefravaret, SykefravaretOppsummert elns.